packages/@webex/plugin-meetings/src/meeting/request.ts

```diff
@@ -792,4 +792,25 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       }
     });
   }
+
+  /**
+   * Make a network request to enable or disable reactions.
+   * @param {boolean} options.enable - determines if we need to enable or disable.
+   * @param {locusUrl} options.locusUrl
+   * @returns {Promise}
+   */
+  toggleReactions({enable, locusUrl, requestingParticipantId}: {enable: boolean, locusUrl: string, requestingParticipantId: string}) {
+    const uri = `${locusUrl}/${CONTROLS}`;
+
+    return this.request({
+      method: 'PUT',
+      uri,
+      body: {
+        reactions: {
+          enabled: enable
```

Comment:
can you explain the process a bit, is there a need to enable reactions? aren't reactions on by default?
i understand we send the reaction "like, smile" via API or websocket. and who can enable this, and when do we enable this in web client?

Reply:
A host/co-host can disable/enable reactions inside the meeting via this settings: 
![image](https://user-images.githubusercontent.com/14194394/211825320-9576a1bc-6928-4504-9095-933abab33b10.png)

So this method is to help do that.
Also, yes, reactions will be enabled by default.

Reply:
aah gotcha, didn't think this was part of settings.